Remora.Commands/Services/CommandService.cs

```diff
@@ -700,25 +659,26 @@ CancellationToken ct
     {
         var materializedParameters = new List<object?>();
 
-        var boundParameters = boundCommandNode.BoundParameters.ToDictionary(bp => bp.ParameterShape.Parameter);
-        var parameterShapes = boundCommandNode.Node.Shape.Parameters.ToDictionary(p => p.Parameter);
+        // Dictionary<ParameterInfo, BoundParameterShape> // parsed, un-typed parameters
```

Comment:
Annotation for what the `var` is

Reply:
I would suggest `var` be used only if there is no ambiguity about the type. Otherwise, it should be fully qualified. I would suggest that over using `var` with a comment.